src/main/java/webApp/webController.java

```diff
@@ -0,0 +1,47 @@
+package webApp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import survey.Survey;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+@Controller
+public class webController {
+    @Autowired
+    private SurveyRepository repo;
+
+    @GetMapping("/index")
+    public String index(Model model){
+        return "index";
+    }
+
+    @PostMapping("/createSurvey")
+    public String createSurvey(@RequestParam("surveyName") String surveyName, @RequestBody ArrayList<Questions> questions) {
+        Survey survey = new Survey(surveyName);
+        survey.setQuestions(questions);
+        repo.save(survey);
+        return "createSurvey";
+    }
+
+    @PostMapping("/retrieveSurvey")
```

Comment:
This should be returning a JSON object, not using the model.

Reply:
Will fix before the merge